.gitignore

```
@@ -40,3 +40,5 @@ yarn-debug.log*
 .yarn-integrity
 
 coverage
+# Ignore vscode settings
+/.vscode/settings.json
```

Pode tirar a pasta toda vai que surge mais arquivos 🤣 

```suggestion
.vscode
```

---

beleza haha